src/components/ArticleCard.jsx

```diff
@@ -13,7 +15,11 @@ export default function ArticleCard(props) {
                     <Card.Text>
                         {article.author} 
                         <br/>
```

Comment:
The use of br here could be an accessibility concern, it would be better to use CSS for styling this card